fastlane/lib/fastlane/runner.rb

```diff
@@ -228,7 +228,11 @@ def execute_action(method_sym, class_ref, arguments, custom_dir: nil, from_actio
           # If another action is calling this action, we shouldn't show it in the summary
           # (see https://github.com/fastlane/fastlane/issues/4546)
 
-          action_name = from_action ? nil : class_ref.step_text
+          if arguments.kind_of?(Array) && arguments.first.kind_of?(Hash)
+            step_name_override = arguments.first[:step_name]
+            arguments.first.delete(:step_name)
```

Comment:
Why delete it?

Reply:
If I don't do this, it will crash because it"s not a supported option of the action

Reply:
though that could make sense to pass it though to the actoin. WDYT?